pywbem_mock/_wbemconnection_mock.py

```diff
@@ -480,6 +481,74 @@ def compile_mof_string(self, mof_str, namespace=None, search_paths=None,
 
         self._merge_repos(mof_repo)
 
+    def compile_dmtf_schema(self, schema_version, schema_root_dir, class_names,
+                            use_experimental=False, verbose=False):
+        """
+        Compile the classes defined by `classes_names` and their dependent
+        classes from the DMTF CIM schema version defined by
+        'schema_version' and keep the downloaded DMTF CIM schema in the
```

Comment:
Use backticks around `schema_version` instead of single quotes.

Reply:
DONE